mkosi/config.py

```diff
@@ -4031,7 +4031,9 @@ def load_environment(args: argparse.Namespace) -> dict[str, str]:
     if gnupghome := os.getenv("GNUPGHOME"):
         env["GNUPGHOME"] = gnupghome
 
-    env |= dict(parse_environment(line) for f in args.environment_files for line in f.read_text().strip().splitlines())
+    env |= dict(parse_environment(line)
+                for f in args.environment_files
+                for line in f.read_text().strip().splitlines())
```

Comment:
I don't mind really but this is completely different from how we format stuff in the rest of the codebase. I know we disagree on proper formatting but let's keep things consistent please.

Reply:
The previous version was a giant completely illegible line… So I think what I had was better. Please see if you like the new version.